### PR TITLE
DOC-5384 Update Kernel Swappiness Recommendations

### PR DESCRIPTION
**Pull request by eric-schneider**
The following documentation is ready for review: [Swap Space and Kernel Swappiness](https://eric-schneider.github.io/DOC-5384-6.0/server/6.0/install/install-swap-space.html)

Documentation Issue: [DOC-5384](https://issues.couchbase.com/browse/DOC-5384)

### CODE REVIEW

**Review by Chaitra19 — COMMENTED** [review]
Ask engineering to approve this. I remember this discussion but this is engineering's call. 
Again this is a generic update, not relevant to beta for 6.5.

**Comment by eric-schneider** [comment]
@Chaitra19 In addition to the recommendation, the _procedure_ on this page needs review for modern operating systems.

**Review by muneebmaster — APPROVED** [review]
Looks good to me

**Review by arod1987 — APPROVED** [review]

File: modules/install/pages/install-swap-space.adoc [hunk 1]
```diff
@@ -2,56 +2,89 @@
...
+On Linux, the kernel's _swappiness_ level indicates how likely the system is to swap pages out of physical memory based on RAM usage.
+Swappiness should be set to 1 or 0 on most Linux systems to achieve optimal Couchbase Server performance.
 
-Couchbase Server efficiently uses available RAM for your working set data; ideally, sufficient RAM remains available to the operating system even with your cluster’s configured Server RAM Quota as well.
-It is always a good idea to configure a reasonable amount of virtual memory or swap space on Linux-based nodes and a Pagefile on Windows.
-Sufficient virtual memory avoids that Couchbase Server processes from being killed by the OS, such as the Linux out of memory (OOM) killer.
+Couchbase Server efficiently uses available RAM for your working set data; ideally, sufficient RAM remains available to the operating system above and beyond your cluster’s configured server RAM quota.
+It's always a good idea to configure a reasonable amount of virtual memory or swap space on Linux-based nodes. (There are no recommended virtual memory optimizations for Windows-based nodes.)
```
Comment (simon-dew) [inline modules/install/pages/install-swap-space.adoc:9]: Any recommendations for macOS?

**Review by simon-dew — APPROVED** [review]
LGTM 👍 — one minor query, not urgent

**Pull request merged**